cli/tests/wpt.jsonc

```diff
@@ -243,13 +243,13 @@
     "jsapi/table/get-set",
     "jsapi/table/toString",
     "webapi/body",
```

Comment:
Body should never have been enabled either.

Reply:
I'm actually unfamiliar with the WPT tests in general, where can I find the source of each individual WPT test?

Reply:
Upstream is web_platform_tests/wpt
Submodule is at test_util/wpt

Reply:
@caspervonb I'm not sure I understood what you had said, where are those directories located?